llama_stack/providers/utils/memory/reranker.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from llama_stack.providers.utils.memory.vector_store import (
+    RERANKER_TYPE_RRF,
+    RERANKER_TYPE_WEIGHTED,
+)
+
+
+class Reranker:
```

Comment:
The code in this file is very much similar to: https://github.com/meta-llama/llama-stack/blob/342550c1e24cc3e354c4afde92940da28ace9a32/llama_stack/providers/inline/vector_io/sqlite_vec/sqlite_vec.py#L68-L122

Can we move that to a common utility so that we can use it for all DBs instead of rewriting it?